privacyidea/api/before_after.py

```diff
@@ -88,6 +88,31 @@ def log_begin_request():
     log.debug("Begin handling of request {!r}".format(request.full_path))
     g.startdate = datetime.datetime.now()
 
+    # remove session from param and gather all parameters, either
+    # from the Form data or from JSON in the request body.
+    ensure_no_config_object()
+    request.all_data = get_all_params(request)
+    privacyidea_server = get_app_config_value("PI_AUDIT_SERVERNAME", get_privacyidea_node(request.host))
+    # Create a policy_object, that reads the database audit settings
+    # and contains the complete policy definition during the request.
+    # This audit_object can be used in the postpolicy and prepolicy and it
+    # can be passed to the innerpolicies.
+    g.policy_object = PolicyClass()
+    g.audit_object = getAudit(current_app.config)
+    # access_route contains the ip addresses of all clients, hops and proxies.
+    g.client_ip = get_client_ip(request,
```

Comment:
we should add the headers here as well, even if they weren't used in the `ttype` endpoint.